models/base/most_recent_course_blocks.sql

```diff
@@ -0,0 +1,30 @@
+{{
+    config(
+        materialized="materialized_view",
+        schema=env_var("ASPECTS_EVENT_SINK_DATABASE", "event_sink"),
+        engine=get_engine("ReplacingMergeTree()"),
+        primary_key=("location"),
+        order_by=("location"),
+        post_hook="OPTIMIZE TABLE {{ this }} {{ on_cluster() }} FINAL",
```

Comment:
I think we might be able to get rid of both the `order_by` and `post_hook` arguments.